sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -169,11 +175,11 @@ public EventHubClientBuilder configuration(Configuration configuration) {
     /**
      * Sets the credential information for which Event Hub instance to connect to, and how to authorize against it.
      *
-     * @param host The fully qualified host name for the Event Hubs namespace. This is likely to be similar to
-     *         {@literal "{your-namespace}.servicebus.windows.net}".
+     * @param host The fully qualified host name for the Event Hubs namespace. This is likely to be similar to {@literal
+     * "{your-namespace}.servicebus.windows.net}".
```

Comment:
These should maintain their indentation. Jonathan had commented that its easier to read when it is indented.